lib/screens/contributors.dart

```diff
@@ -60,206 +66,258 @@ class _ContributorsState extends State<Contributors>
-                                        _launchURL('https://twitter.com/${cardList[index].twitterUsername}');
-                                      },
-                                      child: Row(
-                                        children: [
-                                          SizedBox(
-                                            height: 14,
-                                            width: 14,
-                                            child: Image.network(
-                                              'https://img.icons8.com/fluent-systems-filled/344/twitter.png',
-                                              color: Colors.blueAccent,
-                                            ),
-                                          ),
-                                          const SizedBox(
-                                            width: 5,
-                                          ),
-                                          const Icon(
-                                            Icons.alternate_email,
-                                            size: 14,
-                                            color: Colors.blueAccent,
-                                          ),
-                                          Text(
-                                            cardList[index].twitterUsername,
-                                            style: const TextStyle(
-                                                fontSize: 12,
-                                                color: Colors.blueAccent),
-                                          ),
-                                        ],
-                                      ),
-                                    )
-                                  : const SizedBox(),
-                            ],
-                          ),
-                        ],
-                      ),
-                    ));
-              }),
-      floatingActionButton: const FloatingActionButton(
+                                    ),
+                                  )
+                                : const SizedBox(),
+                          ],
+                        ),
+                      ],
+                    ),
+                  ),
+                );
+              },
+            ),
+      floatingActionButton: FloatingActionButton(
         backgroundColor: Colors.blue,
-        onPressed: addToContributors,
+        onPressed: () {
+          // Trigger the 'Add Contributor' dialog
+          _showAddContributorDialog(context, addContributor);
+        },
         child: Icon(Icons.add),
       ),
     );
   }
 
+  void _showAddContributorDialog(
+      BuildContext context, Function(ContributorCard) addContributorCallback) {
+    TextEditingController usernameController = TextEditingController();
+
+    showDialog(
+      context: context,
+      builder: (context) {
+        return AlertDialog(
+          title: Text('Add Contributor'),

```

Comment:
![image](https://github.com/IEEE-VIT/hacktoberfest-flutter/assets/84137299/d437088d-e7a1-4332-bff6-caa4eb622238)

use const keyword for every widget which avoids unnecessary rebuilds